backend/server.js

```diff
@@ -3,7 +3,7 @@ const swaggerUi = require('swagger-ui-express')
 const swaggerDocument = require('./swaggerConfig')
 
 const app = express()
-const PORT = process.env.PORT || 3001
+const port = 3001; 
```

Comment:
```suggestion
const PORT = process.env.PORT || 3001
```

Por qual motivo mudar o código do backend?